### DISCUSSION TITLE
Customizable Return URL for Spam submissions

### DISCUSSION THREAD

**Discussion by artmem** [solspace/craft-freeform#581] [Feature Requests]
I'm using a return url with a query string to integrate form data with CRM software, but this means all submissions are sent to the CRM, including Spam submissions. It would be helpful if we could specify a different Return URL for submissions that are flagged as spam to prevent this.

**Comment by kjmartens** [comment]
Hi @artmem, just to confirm, are you seeing spammy submissions get flagged as spam AND sent off to the API? It shouldn't be. 🙂

**Reply by artmem** [reply]
Yes. They're flagged as spam in Freeform, but they're sent to the api. My setup is basically setting an api endpoint with a query string of form data as the Return URL in freeform, which then redirects back to my thank you page in Craft.
